contributed_definitions/nyaml/NXsts.yaml

```
@@ -421,23 +413,47 @@ NXsts(NXsensor_scan):
         # parameters how to change the location from measurement to measurement during the scan
         scan_control(NXcollection):
           exists: optional
-          roi(NXtransformations):
+          scan_range(NX_NUMBER):
+            unit: NX_LENGTH
             exists: optional
+            doc: |
+              The scan range (in STM experiment) is the area that the tip will scan. (e.g.
+              5.000000E-9 5.000000E-9)
```

```suggestion
              In STM experiments, the scan range (t) is the area that the tip scans (e.g. 5.000000E-9, 5.000000E-9).
```
If this is always 2D, you should limit its dimension.

---

This is not a 2D data array but a coordinate in 2D space. I mentioned that in doc.